cmd/print.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/jhump/protoreflect/desc"
+	"github.com/jhump/protoreflect/desc/protoprint"
+	stencilv1 "github.com/odpf/stencil/server/odpf/stencil/v1"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/descriptorpb"
+)
+
+// PrintCmd creates a new cobra command for getting .proto files
+func PrintCmd() *cobra.Command {
+	var (
+		req              stencilv1.GetSchemaRequest
+		pathDir          string
+		filterPathPrefix string
+		host             string
+	)
+	cmd := &cobra.Command{
+		Use:   "print",
+		Short: "prints snapshot details into .proto files",
+		Args:  cobra.NoArgs,
+		Annotations: map[string]string{
+			"group:core": "true",
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			conn, err := grpc.Dial(host, grpc.WithInsecure())
+			if err != nil {
+				return err
+			}
+			defer conn.Close()
+			client := stencilv1.NewStencilServiceClient(conn)
+			res, err := client.GetSchema(context.Background(), &req)
+			if err != nil {
+				return err
+			}
+
+			fds := &descriptorpb.FileDescriptorSet{}
+			if err := proto.Unmarshal(res.GetData(), fds); err != nil {
+				return fmt.Errorf("descriptor set file is not valid. %w", err)
+			}
+			fdsMap, err := desc.CreateFileDescriptorsFromSet(fds)
+			if err != nil {
+				return err
+			}
+
+			var filteredFds []*desc.FileDescriptor
+			for fdName, fd := range fdsMap {
+				if filterPathPrefix != "" && !strings.HasPrefix(fdName, filterPathPrefix) {
+					continue
+				}
+				filteredFds = append(filteredFds, fd)
+			}
+
+			protoPrinter := &protoprint.Printer{}
+			if pathDir == "" {
+				for _, fd := range filteredFds {
+					protoAsString, err := protoPrinter.PrintProtoToString(fd)
+					if err != nil {
+						return err
+					}
+					fmt.Printf("\n// ----\n// %s\n// ----\n%s", fd.GetName(), protoAsString)
+				}
+			} else {
```

Comment:
we can remove else condition to make it flat.

Reply:
Just changed the APIs in this command to be compatible with new ones. Haven't touched logic yet. Anyways will look back at this.